tools/dis/dis.cpp

```diff
@@ -31,6 +31,8 @@
 
 #include "spirv-tools/libspirv.h"
 
+#define spvIsInBitfield(value, bitfield) ((value) == ((value)&bitfield))
```

Comment:
I'm not crazy about the copy-paste of identical definition from `spirv_definition.h`, but I suppose it's OK because this is a `.cpp` file.